api_contracts/contracts/award/AwardProfile.md

```diff
@@ -425,9 +420,9 @@ This endpoint returns aggregated award amounts for IDVs.
 ## AwardAmountsResponse (object)
 + award_id: 12178065 (required, number)
     The award id sent in the request.
-+ generated_unique_award_id: `CONT_AW_1540_-NONE-_DJB30605051_-NONE-` (required, string)
++ generated_unique_award_id: `Hello, world!` (required, string)
```

Comment:
Could we make this something like `CONT_AW_1540_NONE_DJB30605051_NONE` to be slightly more realistic? ("Hello, world!" is the string default if you don't provide an example value)